src/Wallabag/CoreBundle/Resources/config/services.yml

```diff
@@ -40,11 +40,40 @@ services:
         class: Graby\Graby
         arguments:
             - { error_message: false }
+            - "@wallabag_core.guzzle.http_client"
+            - "@wallabag_core.graby.config_builder"
         calls:
             - [ setLogger, [ "@logger" ] ]
         tags:
             - { name: monolog.logger, channel: graby }
 
+    wallabag_core.graby.config_builder:
+        class: Graby\SiteConfig\ConfigBuilder
```

Comment:
Sorry I wasn't clear ..
The point was that since we add a custom ConfigBuilder we need to inject the logger otherwise we'll lose information from him. So, yes, add the logger